x-pack/plugins/cases/server/client/cases/find.ts

```diff
@@ -72,6 +75,12 @@ export const find = async ({
       caseOptions: {
         ...queryParams,
         ...caseQueries.case,
+        searchFields:
```

Comment:
let's keep it as a string[] therefore we do not need to convert it back to string[][